src/app/shared/utilities/task-type.ts

```diff
@@ -1,10 +1,5 @@
 export enum TaskType {
-  NON_CLINICAL,
-  CLINICAL,
+  SCHEDULED,
+  ON_DEMAND,
   ALL
```

Comment:
does this need clinical too?

Reply:
This isn't being used anymore. i'm just going to remove it.